### PR TITLE
Add Makefile targets and generated patients for EXM130

### PR DESCRIPTION
**Pull request by lukoktonos**
Related Synthea PR: https://github.com/projecttacoma/synthea/pull/8
JIRA Ticket: https://jira.mitre.org/browse/TACOSTRAT-373

r4 patients can be generated with `make r4`, except for the note directly below:

**Note to reviewers:** In order for the patients to calculate, I needed to checkout `3cd1a29` of `cqf-ruler` (the current HEAD of the master branch as of this PR) and start the server locally.

Before this can be merged
- [ ] After https://github.com/projecttacoma/synthea/pull/8 is merged, update branch in Makefile

### CODE REVIEW

File: EXM_130/Makefile [hunk 1]
```diff
@@ -52,19 +65,45 @@ connectathon:
 	-d @connectathon/fhir3/resources/library/all-libraries-bundle.json
 	touch .seed-cqf-ruler-no-vs
 
+.seed-cqf-ruler-no-vs-r4:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
```
Comment (mgramigna) [inline EXM_130/Makefile]:
```suggestion
	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-r4`; do printf '.'; sleep 5; done
```

File: EXM_130/Makefile [hunk 2]
```diff
@@ -52,19 +65,45 @@ connectathon:
...
+	curl -X PUT \
+  http://localhost:8080/cqf-ruler-dstu3/fhir/Measure/measure-exm130-FHIR4 \
```
Comment (mgramigna) [inline EXM_130/Makefile]:
```suggestion
  http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-exm130-FHIR4 \
```

File: EXM_130/Makefile [hunk 3]
```diff
@@ -52,19 +65,45 @@ connectathon:
...
+  -H 'Content-Type: application/json' \
+	-d @connectathon/fhir3/resources/measure/measure-EXM130_FHIR4-7.2.000.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir/ \
```
Comment (mgramigna) [inline EXM_130/Makefile]:
```suggestion
  http://localhost:8080/cqf-ruler-r4/fhir/ \
```

File: EXM_130/Makefile [hunk 4]
```diff
@@ -52,19 +65,45 @@ connectathon:
...
+  -H 'Content-Type: application/json' \
+	-d @EXM_130_bundle.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir \
```
Comment (mgramigna) [inline EXM_130/Makefile]:
```suggestion
  http://localhost:8080/cqf-ruler-r4/fhir \
```

**Review by mgramigna — CHANGES_REQUESTED** [review]

**Review by mgramigna — APPROVED** [review]

**Comment by okeefm** [comment]
Looks good, branch in `Makefile` just needs to get updated.

**Review by okeefm — APPROVED** [review]
seems reasonable to me 👍

**Pull request merged**